var/spack/repos/builtin/packages/py-bx-python/package.py

```diff
@@ -11,10 +11,15 @@ class PyBxPython(PythonPackage):
     to allow for rapid implementation of genome scale analyses."""
 
     homepage = "https://github.com/bxlab/bx-python"
-    url      = "https://github.com/bxlab/bx-python/archive/v0.7.4.tar.gz"
+    url      = "https://pypi.io/packages/source/b/bx-python/bx-python-0.8.8.tar.gz"
 
-    version('0.7.4', sha256='1066d1e56d062d0661f23c19942eb757bd7ab7cb8bc7d89a72fdc3931c995cb4')
+    version('0.8.8', sha256='ad0808ab19c007e8beebadc31827e0d7560ac0e935f1100fb8cc93607400bb47')
+    version('0.7.4',
+            sha256='1066d1e56d062d0661f23c19942eb757bd7ab7cb8bc7d89a72fdc3931c995cb4',
+            url="https://github.com/bxlab/bx-python/archive/v0.7.4.tar.gz")
 
+    depends_on('python@2.6:2.7,3.3:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
```
At least in 0.8.8. 0.7.4 requires Python 2.4+.